x-pack/plugins/cases/server/client/cases/create.ts

```diff
@@ -84,12 +88,17 @@ export const create = async ({
 
   try {
     try {
-      await auth.ensureAuthorized(query.owner, WriteOperations.Create);
+      await auth.ensureAuthorized(query.owner, Operations.createCase);
     } catch (error) {
-      // TODO: log error using audit logger
+      auditLogger?.log(createAuditMsg({ operation: Operations.createCase, error }));
```

Comment:
Why don't we use our `AuthorizationAuditLogger`?

Reply:
Yeah we could. I was following the alerting team's example here and they have two separate implementations. The `AuthorizationAuditLogger` seemed to only be used within the `Authorization` class. There's no reason we couldn't just make it more generic though.